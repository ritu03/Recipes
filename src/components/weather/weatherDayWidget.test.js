import React from 'react'
import { render, cleanup } from '@testing-library/react'

import WeatherDayWidget from './weatherDayWidget'


const Values = {
  values: {air_pressure: 1017,
    applicable_date: "2021-06-11",
    created: "2021-06-06T03:58:10.877013Z",
    humidity: 66,
    id: 6753910346743808,
    max_temp: 22.875,
    min_temp: 16.345,
    predictability: 75,
    the_temp: 28.74,
    visibility: 9.999726596675416,
    weather_state_abbr: "lr",
    weather_state_name: "Light Rain",
    wind_direction: 149.5.toExponential,
    wind_direction_compass: "SSE",
    wind_speed: 4.778823272090989},
  radioValue: 'Celcius'
}

describe('WeatherDayWidget snapshot', () => {
  afterEach(cleanup)

  test('renders properly', () => {
    const { container } = render(<WeatherDayWidget values={Values} />)
    expect(container).toMatchSnapshot()
  })
})