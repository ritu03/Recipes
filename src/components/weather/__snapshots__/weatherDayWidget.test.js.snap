// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherDayWidget snapshot renders properly 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-2"
  >
    <div
      class="sc-bdnxRM jPuVmM"
    >
      <div
        class="sc-gtsrHT kSJRgO"
      >
        <p />
      </div>
      <img
        height="75"
        src="http://localhost:3000/weatherIcon_undefined.svg"
        width="100"
      />
      <p>
        <span
          class="sc-dlnjwi jezzPh"
        >
          NaN
           
          <span>
             ℉ 
          </span>
        </span>
        <span
          class="sc-hKFxyN iiIRvU"
        >
          NaN
           
          <span>
             ℉ 
          </span>
        </span>
      </p>
      <p>
        Wind Speed : 
        NaN
        kph
      </p>
    </div>
  </div>
</div>
`;
